client/migrations/0017_auto_20200406_2023.py

```diff
@@ -0,0 +1,65 @@
+# Generated by Django 3.0.4 on 2020-04-06 17:23
+
+import django.core.validators
+from django.db import migrations, models
+import django.db.models.deletion
+import multiselectfield.db.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('client', '0016_auto_20200406_1856'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='volunteer',
+            name='email_verified',
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AddField(
+            model_name='volunteer',
+            name='location_address_x',
+            field=models.FloatField(default=0),
+        ),
+        migrations.AddField(
+            model_name='volunteer',
+            name='location_address_y',
+            field=models.FloatField(default=0),
+        ),
+        migrations.AddField(
+            model_name='volunteer',
+            name='score',
+            field=models.IntegerField(default=0),
+        ),
+        migrations.AddField(
+            model_name='volunteer',
+            name='sex',
+            field=models.CharField(choices=[('MALE', 'זכר'), ('FEMALE', 'נקבה')], default=None, max_length=20),
+        ),
+        migrations.AddField(
+            model_name='volunteer',
+            name='wanted_assignments',
+            field=multiselectfield.db.fields.MultiSelectField(choices=[('NIGHBORHOOD_COORDINATOR', 'רכז שכונה'), ('CITY_COORDINATOR', 'רכז עיר'), ('STAFF', 'מטה'), ('HAMAL', 'חמל'), ('PROJECT', 'פרויקט'), ('CHILD_CARE', 'משפחתון'), ('AGRICULTURE', 'חקלאות'), ('MISSIONS', 'משימות')], default=1, max_length=92),
```

Comment:
This is not according to the definition, the choices here represent volunteer type, which is a different field. Please change this to the choices specified in the issue.